### PR TITLE
MAUI-5239 : Prepare UG for Layout customization in .NET MAUI popup

### PR DESCRIPTION
**Pull request by JayashreeRavishankar**
Prepare UG for Layout customization in .NET MAUI popup

### CODE REVIEW

File: MAUI/Popup/layout-customizations.md [hunk 1]
```diff
@@ -0,0 +1,441 @@
...
+<ContentPage xmlns="http://schemas.microsoft.com/dotnet/2021/maui"
+             xmlns:x="http://schemas.microsoft.com/winfx/2009/xaml"
+             xmlns:sfPopup="clr-namespace:Syncfusion.Maui.Popup;assembly=Syncfusion.Maui.Popup"
+             x:Class="PopupMauiLayout.MainPage">
+     <ContentPage.Content>
+        <StackLayout Padding="20">
+            <Button x:Name="clickToShowPopup" Text="ClickToShowPopup" 
+                    VerticalOptions="Start" HorizontalOptions="Center" 
+                    Clicked="ClickToShowPopup_Clicked" />
+            <sfPopup:SfPopup x:Name="sfPopup" AppearanceMode="OneButton" ShowFooter="True">
```
Comment (PiruthivirajM) [inline MAUI/Popup/layout-customizations.md:42]: highlight "AppearanceMode" property line alone.
ensure to highlight the required code in whole doc.

File: MAUI/Popup/layout-customizations.md [hunk 2]
```diff
@@ -0,0 +1,441 @@
...
+The `SfPopup` supports two types of `SfPopup.AppearanceMode`. By default, the `PopupButtonAppearanceMode.OneButton` is set. You can change the appearance by using the `SfPopup.AppearanceMode` property.
+
+The two different appearance modes in the SfPopup are as follows:
+<table>
+<tr>
+<th>Modes</th>
+<th>Description</th></tr>
+<tr>
+<td>`OneButton`</td>
+<td>Shows the `SfPopup` with one button in the footer view. This is the default value.</td></tr>
```
Comment (PiruthivirajM) [inline MAUI/Popup/layout-customizations.md:23]: .... one button (accept button)

File: MAUI/Popup/layout-customizations.md [hunk 3]
```diff
@@ -0,0 +1,441 @@
...
+<tr>
+<td>`TwoButton`</td>
+<td>Shows the `SfPopup` with two buttons in the footer view.</td></tr>
```
Comment (PiruthivirajM) [inline MAUI/Popup/layout-customizations.md:26]: ...two button (accept & decline buttons)

File: MAUI/Popup/layout-customizations.md [hunk 4]
```diff
@@ -0,0 +1,441 @@
...
+
+    private void ClickToShowPopup_Clicked(object sender, EventArgs e)
+    {
+        sfPopup.Show();
+    }
+}
+{% endhighlight %}
+{% endtabs %}
+
+![.NET MAUI Popup with two buttons](Images/layout-customizations/maui-popup-appearance-mode-two-button.png)
```
Comment (PiruthivirajM) [inline MAUI/Popup/layout-customizations.md:111]: [.NET MAUI Popup with accept and decline buttons]

File: maui-toc.html [hunk 5]
```diff
@@ -509,8 +509,8 @@
             <li><a href="/maui/Popup/migration">Migrate from Xamarin.Forms</a></li>
             <li><a href="/maui/Popup/popup-positioning">Popup Positioning</a></li>
             <li><a href="/maui/Popup/popup-size">Popup Size</a></li>
-            <li><a href="/maui/Popup/Popup-Events">Events and Commands</a></li>
-        </ul>
+            <li><a href="/maui/Popup/layout-customizations">Layout Customizations</a></li>
+            <li><a href="/maui/Popup/Popup-Events">Events and Commands</a></li>        </ul>
```
Comment (PiruthivirajM) [inline maui-toc.html:513]: remove unwanted changes showing

File: MAUI/Popup/layout-customizations.md [hunk 6]
```diff
@@ -0,0 +1,451 @@
+---
+layout: post
+title: Layout Customizations in .NET MAUI Popup control | Syncfusion
+description: Learn here all about Layout Customizations support in Syncfusion .NET MAUI Popup (SfPopup) control, its elements and more.
+platform: MAUI
```
Comment (Chrispineagunja) [inline MAUI/Popup/layout-customizations.md:5]: Platform: MAUI

File: MAUI/Popup/layout-customizations.md [hunk 7]
```diff
@@ -0,0 +1,451 @@
+---
+layout: post
+title: Layout Customizations in .NET MAUI Popup control | Syncfusion
+description: Learn here all about Layout Customizations support in Syncfusion .NET MAUI Popup (SfPopup) control, its elements and more.
+platform: MAUI
+control: SfPopup
```
Comment (Chrispineagunja) [inline MAUI/Popup/layout-customizations.md:6]: Control: SfPopup

File: MAUI/Popup/layout-customizations.md [hunk 8]
```diff
@@ -0,0 +1,451 @@
+---
+layout: post
+title: Layout Customizations in .NET MAUI Popup control | Syncfusion
+description: Learn here all about Layout Customizations support in Syncfusion .NET MAUI Popup (SfPopup) control, its elements and more.
+platform: MAUI
+control: SfPopup
+documentation: ug
```
Comment (Chrispineagunja) [inline MAUI/Popup/layout-customizations.md:7]: Documentation: UG

File: MAUI/Popup/layout-customizations.md [hunk 9]
```diff
@@ -0,0 +1,451 @@
...
+platform: MAUI
+control: SfPopup
+documentation: ug
+---
+
+# Layout Customizations in MAUI Popup (SfPopup)
+
+## Popup appearance mode for footer
+
+The `SfPopup` supports two types of `SfPopup.AppearanceMode`. By default, the `PopupButtonAppearanceMode.OneButton` is set. You can change the appearance by using the `SfPopup.AppearanceMode` property.
```
Comment (Chrispineagunja) [inline MAUI/Popup/layout-customizations.md:14]: The `SfPopup` supports two types of `SfPopup.AppearanceMode.` By default, the `PopupButtonAppearanceMode.OneButton` is set. Change the appearance by using the `SfPopup.AppearanceMode` property.

File: MAUI/Popup/layout-customizations.md [hunk 10]
```diff
@@ -0,0 +1,451 @@
...
+    {
+        sfPopup.Show();
+    }
+}
+{% endhighlight %}
+{% endtabs %}
+
+![.NET MAUI Popup with accept button](Images/layout-customizations/maui-popup-appearance-mode-one-button.png)
+
+In the following code example, set the `SfPopup.AppearanceMode` property as `TwoButton` which displays both Accept button and Decline button in the footer view.
```
Comment (Chrispineagunja) [inline MAUI/Popup/layout-customizations.md:73]: In the following code example, set the `SfPopup.AppearanceMode` property as `TwoButton,` which displays both Accept and Decline buttons in the footer view.

File: MAUI/Popup/layout-customizations.md [hunk 11]
```diff
@@ -0,0 +1,451 @@
...
+<th>Description</th></tr>
+<tr>
+<td>`OneButton`</td>
+<td>Shows the `SfPopup` with one button (accept button) in the footer view. This is the default value.</td></tr>
+<tr>
+<td>`TwoButton`</td>
+<td>Shows the `SfPopup` with two buttons (accept & decline buttons) in the footer view.</td></tr>
+</table>
+
+In the following code example, set the `SfPopup.AppearanceMode` property as `OneButton` which displays only the Accept button in the footer view.
```
Comment (Chrispineagunja) [inline MAUI/Popup/layout-customizations.md:29]: In the following code example, set the `SfPopup.AppearanceMode` property as `OneButton,` which displays only the Accept button in the footer view.

File: MAUI/Popup/layout-customizations.md [hunk 12]
```diff
@@ -0,0 +1,451 @@
...
+{% endhighlight %}
+{% endtabs %}
+
+![.NET MAUI Popup with accept and decline buttons](Images/layout-customizations/maui-popup-appearance-mode-two-button.png)
+
+## Popup region customization
+
+### Disable header
+
+You can display the Popup without header by using the property `SfPopup.ShowHeader`. Default value is true. Find the code example of the same below.
```
Comment (Chrispineagunja) [inline MAUI/Popup/layout-customizations.md:121]: Display the Popup without a header by using the property `SfPopup.ShowHeader`. The default value is true. Find the code example of the same as follows.

File: MAUI/Popup/layout-customizations.md [hunk 13]
```diff
@@ -0,0 +1,451 @@
...
+    }
+}
+{% endhighlight %}
+{% endtabs %}
+
+![.NET MAUI Popup without header](Images/layout-customizations/maui-popup-without-header.png)
+
+### Enable footer
+
+You can display the Popup with footer by using the property, `SfPopup.ShowFooter`. Default value is false. Find the code example of the same below.
```
Comment (Chrispineagunja) [inline MAUI/Popup/layout-customizations.md:166]: Display the Popup with the footer by using the `SfPopup.ShowFooter` property.  The default value is false. Find the code example of the same as follows.

File: MAUI/Popup/layout-customizations.md [hunk 14]
```diff
@@ -0,0 +1,451 @@
...
+    }
+}
+{% endhighlight %}
+{% endtabs %}
+
+![.NET MAUI Popup with footer](Images/layout-customizations/maui-popup-with-footer.png)
+
+## Enable close icon
+
+You can show the Popup with close icon by using the property, `SfPopup.ShowCloseButton`. Default value is false. Find the code example of the same below.
```
Comment (Chrispineagunja) [inline MAUI/Popup/layout-customizations.md:209]: Show the Popup with a close icon by using the `SfPopup.ShowCloseButton` property. The default value is false. Find the code example of the same as follows.

**Review by PiruthivirajM — APPROVED** [review]
approved

**Pull request merged**